dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/Parent.java

```diff
@@ -131,7 +131,11 @@ protected void loadDockerfileGithubUtil(DockerfileGitHubUtil _dockerfileGitHubUt
                 // fork the parent if not already forked
                 if (!parentReposForked.contains(parentRepoName)) {
                     log.info("Forking {}", parentRepoName);
-                    dockerfileGitHubUtil.closeOutdatedPullRequestAndFork(parent);
+                    GHRepository fork = dockerfileGitHubUtil.closeOutdatedPullRequestAndFork(parent);
+                    if (fork == null) {
+                        log.info("Could not fork {}", parentRepoName);
+                        pathToDockerfilesInParentRepo.remove(parentRepoName, c.getPath());
+                    }
                     parentReposForked.add(parentRepoName);
```

Comment:
Should we be adding to `parentReposForked` if this repo was not forked?

Reply:
parentReposForked is not used outside of this method. It will avoid trying to create fork and failing multiple times if we are updating multiple Dockerfiles in a same fork.

Reply:
Might be nice to rename this since we may not have forked all of the repos in this list. Possibly `parentReposForked`->`parentReposAlreadyChecked`. Either that, or change the comment above https://github.com/salesforce/dockerfile-image-update/pull/48/files#diff-da5f0732ed5f695522bdefddf18a2555R132 to say something like `fork the parent if not already forked or we couldn't fork`

Reply:
Updated PR